atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -44,6 +44,28 @@
 
     @Value.Default
     public int poolSize() {
+        return 30;
+    }
+
+    /**
+     * The proportion of {@link #poolSize()} connections that are checked approximately
+     * every {@link #timeBetweenConnectionEvictionRunsSeconds()} seconds to see if has been idle at least
+     * {@link #idleConnectionTimeoutSeconds()} seconds and evicts it from the pool if so. For example, given the
+     * the default values, 0.1 * 30 = 3 connections will be checked approximately every 20 seconds and will
+     * be evicted from the pool if it has been idle for at least 10 minutes.
+     */
+    @Value.Default
+    public double proportionConnectionsToCheckPerEvictionRun() {
+        return 0.1;
```

Comment:
probably need to `@Check` this isn't set to 0 and error out in `setNumTestsPerEvictionRun`